docs/realm.js

```diff
@@ -277,6 +277,11 @@ Realm.defaultPath;
  *   will be skipped if another process is accessing it.
  * @property {string} [path={@link Realm.defaultPath}] - The path to the file where the
  *   Realm database should be stored.
+ * @property {boolean} [inMemory=false] - Specifies if this Realm should be opened in-memory. This
+ *    still requires a path (can be the default path) to identify the realm so other processes can
+ *    open the same realm. The file will also be used as swap space if the realm becomes bigger than
+ *    what fits in memory, but it is not persistent and will be removed when the last instance
```

Comment:
"persistent" -> "persisted"

Reply:
@kneth Please just fix clear spelling mistakes. That's faster than mentioning them :-)